common/lib/xmodule/xmodule/css/video/_reference.scss

```diff
@@ -0,0 +1,800 @@
+& {
+    margin-bottom: ($baseline*1.5);
+}
+
+// div.video {
```

Comment:
This file was used for the old skin. While building the new UI, I referenced it often as some of the same styles were kept. This file is here mainly for reference, but once this project is complete and merged, this file can probably be deleted safely.